subprojects/core/src/main/java/org/gradle/api/file/CopySpec.java

```diff
@@ -147,6 +147,28 @@
     CopySpec filesMatching(String pattern, Action<? super FileCopyDetails> action);
 
     /**
+     * Configure the {@link org.gradle.api.file.FileCopyDetails} for each file whose path matches any of the specified Ant-style patterns.
+     * This is equivalent to using eachFile() and selectively applying a configuration based on the file's path.
+     *
+     * @param patterns Ant-style patterns used to match against files' relative paths
+     * @param action action called for the FileCopyDetails of each file matching pattern
+     * @return this
+     */
+    @Incubating
+    CopySpec filesMatching(String[] patterns, Action<? super FileCopyDetails> action);
```

Comment:
Please use `String...` instead of `String[]`. This is true for all the other methods in this class, too.